pkgs/by-name/as/astro-language-server/package.nix

```diff
@@ -2,35 +2,35 @@
   lib,
   stdenv,
   fetchFromGitHub,
-  pnpm_8,
+  pnpm_9,
```

Comment:
@jfvillablanca I know you just following existing pattern, but such overrides are preferred to be done in all-packages.nix

https://github.com/NixOS/nixpkgs/tree/master/pkgs/by-name#changing-implicit-attribute-defaults

Reply:
hello, can you clarify your suggestion?
the `astro-language-server` is not imported in `all-packages.nix` since it uses `by-name`